paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/FlinkCatalog.java

```diff
@@ -575,7 +582,26 @@ public static Schema fromCatalogTable(CatalogTable catalogTable) {
     private static Map<String, String> getColumnComments(CatalogTable catalogTable) {
         return catalogTable.getUnresolvedSchema().getColumns().stream()
                 .filter(c -> c.getComment().isPresent())
-                .collect(Collectors.toMap(UnresolvedColumn::getName, c -> c.getComment().get()));
+                .collect(
+                        Collectors.toMap(
+                                UnresolvedColumn::getName,
+                                c -> {
+                                    String comment = c.getComment().get();
+                                    if (comment.toUpperCase().startsWith("U&'")) {
+                                        try {
+                                            SqlParser parser = SqlParser.create(comment + "'");
```

Comment:
We need test to cover chinese comments, for Flink, for Hive

Reply:
https://github.com/apache/incubator-paimon/blob/fbff232ac43c6e19079c0d0c373601a3a54871f2/paimon-hive/paimon-hive-connector-common/src/test/java/org/apache/paimon/hive/HiveCatalogITCaseBase.java#L247-L263